app/services/WebPlayer.java

```diff
@@ -0,0 +1,26 @@
+package services;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class WebPlayer implements Player {
+
+    private Marks mark;
+
+    public WebPlayer(Marks mark) {
+        this.mark = mark;
+    }
+
+    public Marks getMark() {
+        return mark;
+    }
+
+    public Integer getLocation(Board board) throws Exception {
+        return null;
+    }
+
+    public Class playerType() {
+        return getClass();
+    }
+}
```

Comment:
I think this is a LSP violation.  Because my HumanPlayer takes an Input as an argument to the constructor, and I couldn't work out what that Input would be for a web game, I made a new player.  I also tried making a WebInput class that implements my Input interface that I could pass into the constructor, but again, it wasn't really doing anything and would never be used.  

In the core, the Input interface checks the validity of the chosen move, formats it (converts string to int and -1 for the board array), but as I'm using button inputs with set values, choosing an invalid location is impossible.

I think the WebInput that can be passed to the HumanPlayer is better than this WebPlayer, but it's still an LSP violation and the HumanPlayer methods would not really be used, so any advice would be great!